testsuite/shared/src/main/java/org/wildfly/test/distribution/validation/AbstractValidationUnitTest.java

```diff
@@ -152,6 +154,7 @@ public class AbstractValidationUnitTest {
             final File schemaDir = new File(JBOSS_DIST_DIR, SCHEMAS_LOCATION);
 
             final File[] xsds = schemaDir.listFiles(new SchemaFilter(EXCLUDED_SCHEMA_FILES.toArray(new String[0])));
+            Arrays.sort(xsds);
```

Comment:
@alexbadia1 How does this address the JIRA description statement that:

"the order of the `SCHEMA_SOURCES` matters since some schemas may be dependent on each other, (i.e. one schema importing another)" ?

It makes the order deterministic, but I'm not sure why lexicographical ordering of path names results in avoiding incorrect orders between dependent schemas.

I think your PR's fine; being deterministic is good, and if it passes we know the lexicographical sort order is an ok won to stick to.